packages/integration-sdk-cli/src/utils/getSortedJupiterOneTypes.test.ts

```diff
@@ -2,197 +2,215 @@ import {
   RelationshipClass,
   RelationshipDirection,
   Step,
+  StepEntityMetadata,
+  StepMappedRelationshipMetadata,
+  StepRelationshipMetadata,
 } from '@jupiterone/integration-sdk-core';
 import { collectGraphObjectMetadataFromSteps } from './getSortedJupiterOneTypes';
+import { randomUUID } from 'crypto';
+
+function createIntegrationStep({
+  entities = [],
+  relationships = [],
+  mappedRelationships = [],
+}: {
+  entities?: StepEntityMetadata[];
+  relationships?: StepRelationshipMetadata[];
+  mappedRelationships?: StepMappedRelationshipMetadata[];
+}): Step<any> {
+  const stepId = randomUUID();
```

Comment:
FYI if we do this we need to add an engine field in the `package.json`. I believe this was added in 14.19. Correct me if I'm wrong.

Reply:
Good comms. Going to use `uuid` and we can make the breaking change in a separate PR.